src/Main.java

```diff
@@ -1,9 +1,36 @@
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
+        Scanner userInput = new Scanner(System.in);
+        Integer [] numeric = { 1,2,3,4,5,6,7,8,9,0};
+        String [] alphabetic = {"een", "twee", "drie", "vier", "vijd", "zes", "zeven", "acht", "negen", "nul"};
+        Translator firstTry = new Translator(alphabetic, numeric);
+        boolean play = true;
+        String ongeldig = "Ongeldige invoer";
 
-        // Hier mag je je code schrijven voor de hoofd-opdracht
-
+        while(play)
+        {
+            System.out.println("Input a value between 0 and 9");
+            int userOption = userInput.nextInt();
+            System.out.println("Your input is translated to " + firstTry.Translate(userOption));
```

Comment:
Probeer voor de duidelijkheid van de gebruiker niet te veel te wisselen in voertaal.